core/esmf-aspect-model-resolver/src/test/resources/samm_1_0_0/org.eclipse.esmf.test/2.0.0/BammTest.ttl

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH
+#
+# See the AUTHORS file(s) distributed with this work for additional
+# information regarding authorship.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+
+@prefix bamm:   <urn:bamm:io.openmanufacturing:meta-model:1.0.0#> .
+@prefix bamm-c: <urn:bamm:io.openmanufacturing:characteristic:1.0.0#> .
+@prefix bamm-e: <urn:bamm:io.openmanufacturing:entity:1.0.0#> .
+@prefix :       <urn:bamm:org.eclipse.esmf.test:2.0.0#> .
+@prefix unit:   <urn:bamm:io.openmanufacturing:unit:1.0.0#> .
+@prefix xsd:    <http://www.w3.org/2001/XMLSchema#> .
+
+:BammTest a bamm:Aspect ;
+   bamm:name "BammTest" ;
+   bamm:preferredName "BAMM Test"@en ;
+   bamm:properties ( :property ) ;
+   bamm:operations ( ) .
+
+:property a bamm:Property ;
+   bamm:name "TestProperty" ;
```

Comment:
The value of `bamm:name` must be identical to the URN's local part, i.e. in this case "property".

```suggestion
   bamm:name "property" ;
```